### PR TITLE
Fix auto geolocation enabled on the facility locator

### PR DESCRIPTION
**Pull request by brangi**
## Description

MapBox SDK was upgraded and the latest version is running in staging environment (https://staging.va.gov/find-locations). Currently, there is an issue when calling `reverseGeocode` that causes the auto detection of geolocation. This fix should address the following issues:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/4735
https://github.com/department-of-veterans-affairs/va.gov-team/issues/4736
https://github.com/department-of-veterans-affairs/va.gov-team/issues/4734

## Testing done
Locally

## Acceptance criteria
- [ ]Fix geolocation detection (when allowing location in the dialog)

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/facility-locator/containers/VAMap.jsx [hunk 1]
```diff
@@ -333,7 +333,7 @@ class VAMap extends Component {
         .catch();
     } else {
       /**
-       * Current SDk uses geocodeForward fn to make the API all (callback)
+       * Current SDk uses geocodeReverse fn to make the API all (callback)
```
Comment (kelsonic) [inline src/applications/facility-locator/containers/VAMap.jsx]: Did we mean `call` here in the comment? 🙂`API all` -> `API call`

File: src/applications/facility-locator/containers/VAMap.jsx [hunk 2]
```diff
@@ -302,8 +302,8 @@ class VAMap extends Component {
        * New SDk uses forwardGeocode fn to make the API call(promise)
```
Comment (kelsonic) [inline src/applications/facility-locator/containers/VAMap.jsx]: `forwardGeocode` -> `reverseGeocode`

File: src/applications/facility-locator/tests/mapbox/mapbox-api-services.unit.spec.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+import { mapboxToken } from '../../utils/mapboxToken';
+
+const mapboxClient = MapboxClient({ accessToken: mapboxToken });
+
+const mbxGeo = require('@mapbox/mapbox-sdk/services/geocoding');
+
+const mbxClient = mbxGeo(mapboxClient);
+
+import { BOUNDING_RADIUS } from '../../constants';
```
Comment (kelsonic) [inline src/applications/facility-locator/tests/mapbox/mapbox-api-services.unit.spec.js]: This is a somewhat trivial ask, but could we group the `import` statements together for readbility? Perhaps something like this:

```js
// Node modules.
import { expect } from 'chai';
import MapboxClient from '@mapbox/mapbox-sdk';
import mbxGeo from '@mapbox/mapbox-sdk/services/geocoding';
// Relative imports.
import { BOUNDING_RADIUS } from '../../constants';
import { mapboxToken } from '../../utils/mapboxToken';


const mapboxClient = MapboxClient({ accessToken: mapboxToken });
const mbxClient = mbxGeo(mapboxClient);
```

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**